### PR TITLE
PMM-5133: node type inventory

### PR DESCRIPTION
**Pull request by lunaticusgreen**
https://github.com/Percona-Lab/pmm-submodules/pull/576

### CODE REVIEW

**Comment by lunaticusgreen** [comment]
image

File: pmm-app/src/pmm-inventory/panel.tsx [hunk 1]
```diff
@@ -94,6 +94,11 @@ export const nodesColumns = [
     dataIndex: 'node_id',
     key: 'node_id',
   },
+  {
+    title: 'Node Type',
```
Comment (BupycHuk) [inline pmm-app/src/pmm-inventory/panel.tsx]: looks like something wrong here. Title is `Node Type`, but key and dataIndex are `agentType`

Comment (lunaticusgreen) [reply pmm-app/src/pmm-inventory/panel.tsx]: This is just a common name for this type of column, all tables have similar structure

Comment (BupycHuk) [reply pmm-app/src/pmm-inventory/panel.tsx]: maybe it's better to rename it then. looks really confusing.

**Pull request merged**